lib/pages/leaderboard/podiumSlot.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class PodiumSlot extends StatelessWidget {
+  PodiumSlot({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(alignment: Alignment.bottomCenter, children: <Widget>[
+      CircleAvatar(
+        radius: 40,
+        backgroundImage: AssetImage('assets/raysmall.png'),
+      ),
+      Icon(Icons.looks_two)
```

Comment:
I probably need to change the display underneath the backgroundImage to some sort of container so that we can hold the display name and the rank.

Reply:
Ya sure, we can revisit this